orttraining/orttraining/core/graph/optimizer/sgd_optimizer_builder.cc

```diff
@@ -35,7 +57,8 @@ Status SGDOptimizerBuilder::Build(
     std::vector<TensorProto>& /* new_external_initializers */,
     std::vector<ArgDef>& output_weight_argdefs,
     std::vector<ArgDef>& output_gradient_argdefs,
-    bool /* enable_grad_clipping */) const {
+    bool /* enable_grad_clipping */,
+    const NameMLValMap /* shared_optim_state */) const {
```

Comment:
Does ```NameMLValMap``` need to be passed in by value instead of a const ref?

Reply:
it should be a reference, changing